routes/auth/auth.js

```diff
@@ -118,4 +118,18 @@ router.post("/login", async (req, res) => {
   }
 });
 
+router.delete("/logout", (req, res) => {
+  if (req.session) {
```

Comment:
thats wrong. you're not using sessions anymore...
This is WRONG!

you should be clearing the cookie... I went over this with you

Reply:
Fixed